.github/workflows/gradle.yml

```diff
@@ -0,0 +1,39 @@
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        java-version:
+          - 8
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          show-progress: false
+          fetch-depth: 0
+
+      # Step to set up JDK
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          java-version: ${{ matrix.java-version }}
+          distribution: 'temurin'
+          cache: gradle
+
+      # Step to set up gradle.properties
+      - name: Set up gradle.properties
+        run: |
+          cp gradle.properties.example gradle.properties
+          grep -q '^skipSigning=' gradle.properties || echo "skipSigning=false" >> gradle.properties
```

Comment:
I don't think you need the `cp` or `grep` here. I think it can just be as simple as:

```
echo "skipSigning=false" >> gradle.properties
```

Reply:
> I don't think you need the `cp` or `grep` here. I think it can just be as simple as:
> 
> ```
> echo "skipSigning=false" >> gradle.properties
> ```

made the suggested changes.